src/components/MenuButtonView.tsx

```diff
@@ -17,12 +19,17 @@ export default function MenuButtonView (props: { username: string, spaceIsClosed
 
   const logOut = () => {
     const prompt = confirm('Are you sure you want to log out?')
+    // TODO: When a user disconnects via the /disconnect endpoint, it still shows that user as in the room. I suspect
```

Comment:
Have you tried this recently / with the latest changes from main? I pushed some changes on Wednesday 9/22 that I would have expected to fix this, so I'm curious if they're still persisting.